internal/batches/workspace/workspace.go

```diff
@@ -95,9 +95,12 @@ func BestCreatorType(ctx context.Context, images map[string]docker.Image) Creato
 	// if you have a batch change with steps that run as UID 1000 and then UID
 	// 2000, you'll get errors when the second step tries to write.
 
-	// For the time being, we're only going to consider volume mode on Intel
-	// macOS.
-	if runtime.GOOS != "darwin" || runtime.GOARCH != "amd64" {
+	// NOTE: For the time being, we're only going to consider volume mode on Intel or M1
+	// macOS. As we've generally not observed many issues from users using volume mode, and
+	// it is the faster of the options on non-Linux platforms (and only a touch slower on
+	// Linux itself), we should consider making it the default in the future for the sake of
+	// consistency.
+	if runtime.GOOS != "darwin" || (runtime.GOARCH != "amd64" && runtime.GOARCH != "arm64") {
```

Comment:
Maybe we should just remove the `GOARCH` checks altogether at this point, since macOS doesn't run anywhere else anyway.

Reply:
👍

Reply:
I kinda like keeping it in to make things explicit, juuuuuust in case someone comes in with a weird franken-machine that somehow is running on a different architecture. 😂

Reply:
I don't think Apple would ever distribute other binaries, but doesn't matter much to me 😁

Reply:
Yeah, I'm not too fussed, since I think we'll probably remove this check altogether soon, but I also think it's unlikely to be an issue in practice if we don't have it.

Reply:
sure sure. 🙂 I'll just leave it for now since, as Adam mentions, we'll probably remove this check entirely the next time we're thinking about it, anyway.